04_numeronym/kasunfdo/main_test.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote(`[a11y K8s abc]
+`)
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+func TestNumeronyms(t *testing.T) {
+	actual := numeronyms("testString", "www", "W3school")
+	expected := []string{"t8g", "www", "W6l"}
+	for i, v := range actual {
+		if expected[i] != v {
+			t.Errorf("Unexpected output. Expected: %q - Actual: %q", expected[i], v)
+		}
+	}
+}
+
+func TestGetNumeronyms(t *testing.T) {
```

Comment:
This should really be called `TestGetNumeronym` (singular, not plural) as that is the name of the function being tested. But if you do rename the function as I suggest, don't forget to rename this test function too.